forest/daemon/src/daemon.rs

```diff
@@ -392,6 +395,27 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
     db
 }
 
+// returns the first error with which any of the services end
+// in case all services finished without an error sleeps for more than 2 years
+// and then returns with an error
```

Comment:
this is not true, no? I mean, if I understand the code correctly, then if all services finished without an error then nothing is returned, ever. Or am I missing something?

Reply:
Yes, we are waiting for CTRL-C to be pressed, otherwise we are waiting indefinitely